WTWSMS/events/ze_lord_spiritual_events.txt

```diff
@@ -29,7 +29,6 @@ character_event = {
 			independent = yes
 			higher_tier_than = DUKE
 		}
-		proselyte_religion_trigger = yes
```

Comment:
It _is_ possible to set it up by a Rule. Simply have the Demand Religious Converstion Targetted Decisions's POTENTIAL include all religions that are not unreformed Pagans (and likewise with the events and) if the Rule is set to match the Vanilla rendition,a nd use your complex system otherwise.

(By the way, something is wrong with it: I wasn't able to convert a landed Tengri man with positive one hundred Opinion, despite the fact that he was not Zealous or Cynical.)